src/components/global-header/global-header.component.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { PaddingProps, FlexboxProps } from "styled-system";
+import { ThemeObject } from "style/themes/base";
+import { baseTheme } from "../../style/themes";
+import StyledNavigationBar from "../navigation-bar/navigation-bar.style";
+import Box from "../box";
+
+export interface GlobalHeaderProps extends PaddingProps, FlexboxProps {
+  /** Child elements */
+  children?: React.ReactNode;
+  /** Logo to render */
+  logo?: React.ReactNode;
+}
+
+interface StyledGlobalHeaderProps {
+  theme: ThemeObject;
+}
+const StyledGlobalHeader = styled(StyledNavigationBar)<StyledGlobalHeaderProps>`
```

Comment:
Thanks for raising this @robinzigmond. I agree we should be keeping things consistent, but given this file is not very big like you said, I think this should be sufficient. This would be similar to how we have structured `Box` which is also not very big 😄